xbmc/filesystem/NFSFile.cpp

```diff
@@ -336,13 +337,21 @@ bool CNfsConnection::Connect(const CURL& url, std::string &relativePath)
 
     if (m_readChunkSize == 0)
     {
-      CLog::Log(LOGDEBUG, "NFS Server did not return max read chunksize - Using 128K default");
-      m_readChunkSize = 128 * 1024; // 128K
+      const auto settings = CServiceBroker::GetSettingsComponent()->GetSettings();
+      const int chunkSize = settings ? settings->GetInt(SETTING_NFS_CHUNKSIZE) * 1024 : 128 * 1024;
+
+      CLog::Log(LOGDEBUG, "NFS Server did not return max read chunksize - Using setting value {}",
+                chunkSize);
+      m_readChunkSize = chunkSize;
     }
     if (m_writeChunkSize == 0)
     {
-      CLog::Log(LOGDEBUG, "NFS Server did not return max write chunksize - Using 128K default");
-      m_writeChunkSize = 128 * 1024; // 128K
+      const auto settings = CServiceBroker::GetSettingsComponent()->GetSettings();
```

Comment:
I think this might be enough in practice. What we do not handle is: NFS server says 256KB and we wanted to use a lower value, cause the user only watches BD ISOs.

May be, but I leave that to you: Changing the positive log path, e.g. Server returned a max chunk value - ignoring user defined value?

Reply:
Handled this case: value returned by server is bigger that client setting, then is used the smallest of two.

Reply:
Thanks!